Catch up Chapters/7-SpatialMapping-Completed/Assets/Scripts/SphereCommands.cs

```diff
@@ -29,7 +29,8 @@ void OnReset()
         var rigidbody = this.GetComponent<Rigidbody>();
         if (rigidbody != null)
         {
-            DestroyImmediate(rigidbody);
+            rigidbody.isKinematic = true;
```

Comment:
In order to not reintroduce the bug DestroyImmediate was used to fix, I turned isKinematic on to stop the effect of gravity. Moving the position change before DestroyImmediate didn't fix #14 like I thought.